### ISSUE TITLE
[FR] Revolut Bank BIC is Missing

### ISSUE THREAD

**Issue by BenjaminKasprzak** [mdomke/schwifty#135]
The Revolut Bank, with BIC `REVOFRP2 ` is missing on the latest version of the package.
Here an example of a such IBAN : `FR7628233000018465152778611`

As we can see on `https://fr.iban.com/iban-checker`, it's a good Revolut IBAN ! 

<img width="726" alt="Capture d’écran 2023-02-08 à 11 54 05" src="https://user-images.githubusercontent.com/72394628/217509874-b654f1aa-bcde-4cda-90a7-d03ed100c5a0.png">

Ping @mdomke

**Comment by BenjaminKasprzak** [comment]
Ping ? 🤔

**Comment by mdomke** [comment]
Fixed in `2024.1.1.post0`
```pycon
>>> from schwifty import IBAN
>>> IBAN("FR7628233000018465152778611").bic
<BIC=REVOFRP2>
```
